[RPHAM-3158] Updating documentation (#2899)
kiegroup/kie-docs: doc-content/shared-kie-docs/src/main/asciidoc/KieServer/controller-java-api-con.adoc

To view configured user roles for your {PRODUCT} distribution, navigate to `~/$SERVER_HOME/standalone/configuration/application-roles.properties` and `~/application-users.properties`.

> Assuming a Keystore has been already setup, to add a user with the `kie-server` role or the `rest-all` role or both, navigate to `~/$SERVER_HOME/bin` and run the following command with the role or roles specified:

Reviewer: ```suggestion
To add a user with the `kie-server` role or the `rest-all` role or both (assuming a Keystore is already set), navigate to `~/$SERVER_HOME/bin` and run the following command with the role or roles specified:
```

[source,bash]
----
$ ./add-user.sh -a --user <USERNAME> --password <PASSWORD> --role kie-server,rest-all
----

> if Keystore has not  been already setup, you will need to create it by executing following command

Reviewer: ```suggestion
In case the Keystore is not set, then execute the following command to create a Keystore:
```

[source,bash]
----
$ keytool -importpassword -keystore $SERVER_HOME/standalone/configuration/kie_keystore.jceks -keypass <SECRETKEYPASSWORD> -alias kieserver -storepass <SECRETSTOREPASSWORD> -storetype JCEKS
----

> and add following properties to `~/$SERVER_HOME/standalone/configuration/standalone-full.xml`

Reviewer: ```suggestion
Also, add the following properties to `~/$SERVER_HOME/standalone/configuration/standalone-full.xml`:
```
